GNUmakefile

```diff
@@ -110,11 +110,18 @@ go-solidity-wrappers-functions: pnpmdep abigen ## Recompiles solidity contracts
 generate: abigen codecgen mockery ## Execute all go:generate commands.
 	go generate -x ./...
 
+
+DIR_ARGS ?=-recurse=true
+
+## Usage:
+##   - `make testscripts`: runs all txtar scripts in testdata/scripts, recursively by using the default DIR_ARGS flag.
+##   - `make testscripts DIR_ARGS="-dir=./testdata/scripts"`: runs just the txtar scripts in the testdata/scripts dir.
+##   - `make testscripts DIR_ARGS="-dir=./testdata/scripts/node -recurse=true"`: runs the txtar scripts in the testdata/scripts/node dir, recursively.
```

Comment:
Hmmm I wonder if this is trying to do too much at once. The non recursive case doesn't need to call the `lstxtardirs` program since testscript already works that way. For that reason I was imaging we'd have separate make command for directly invoking testscript once that we could call from this recursive command. :thinking:

Reply:
Ah got it! I think I misunderstood your intent from our slack convo. So if I understand you, you're suggesting that we'd have a:
- `make testscripts -dirs="DIR_A DIR_B` command that runs non-recursively on just the given dirs
- and a `make testscripts-all` command that pipes the result of lstxtardirs into this child command?

How should testscripts-update be adapted? Should it point to `make testscripts` or the -all variant?

Reply:
I'm not sure what the best approach is since I don't grok Make well enough to know the options. Maybe this is better left for a follow up? Or @HenryNguyen5 might have some ideas for passing params with a smoother UX.

Reply:
We could also use a tools/ script at the lowest level, and just call that direction for the complicated cases so we don't have to adapt Make to forward things.

Reply:
Yep, a follow up sounds good! In that case, what I'll do is leave the flags available for the cmd, but not use them at the Makefile level (i.e. preserve exactly the existing API, no Make flags passed in and just using the new cmd to list the dirs).

Reply:
Sounds good :+1: Should we remove these comments for now?
```suggestion
```